wolfcrypt/src/rsa.c

```diff
@@ -613,7 +617,8 @@ static int RsaPad_PSS(const byte* input, word32 inputLen, byte* pkcsBlock,
     ret = RsaMGF(mgf, h, hLen, pkcsBlock, pkcsBlockLen - hLen - 1, heap);
     if (ret != 0)
         return ret;
-    pkcsBlock[0] &= 0x7f;
+    /* TODO: use the number of bits in the prime */
```

Comment:
Message will not verify if the number of bits is wrong.
Added code to fix this.